static/assets/js/vendors/webauthn.js

```diff
@@ -0,0 +1,131 @@
+// Copyright (c) 2017 Duo Security, Inc. All rights reserved.
```

Comment:
do you know if this file can be installed via `npm`? Even if it's quite small, updating it via npm is a bit easier :)

Reply:
I don't think so. [They](https://github.com/duo-labs/py_webauthn/) didn't publish this as an npm package but as a pip one.

An alternative is using [Github's solution](https://github.com/github/webauthn-json). Unfortunately, I found this project after finished all these works so I didn't use it in the current version. This one may not fully compatible with the previously mentioned python package, so probably need some extra works. 

Let me know how you wish to handle this!

Reply:
Do you think the Github one offers the same features as the Duo-Labs one? It seems to me that packaging on Github repo is proper and seems to be more maintained. The Duo-Labs one looks more like a proof-of-concept for me but that might be just a feeling.

Reply:
I'm not sure if they encoded the data in the same way...

Reply:
I see, let's stay with the current way then.